documents/manual_tests.txt

```diff
@@ -140,13 +140,12 @@ Map Page
         () When info windows overlap, the one that is opened later should always be on top
         () When info windows overlap, clicking on the one behind should bring it forward
         () User provided name shows up.
-        () Correct links show up: 'zoom in', 'circle off', 'add to route', 'details', 'remove',
-        () Links
+        () Correct links show up:
             () zoom in
             () circle on
             () add to route
-            () details
-                () should show 'GPS' field only.
+            () details/hide
+                () should toggle 'GPS' field only.
```

Comment:
It toggles more than just the GPS field now, right? More broadly, I feel like the interaction between summary, details, and history could be more intuitive. I don't have an alternative proposal yet though :-) so I'm inclined to say it's worth getting the functionality at least out to test, if not prod.

Reply:
Yeah it does haha. I think that this section is supposed to be about custom markers, but to be fair this document could use another refresher since things have changed since I modified it in #49 almost 3 years ago: https://github.com/supercharge-info/supercharge.info/pull/49/commits/628311ec47f6a13379f1af5279ff97c27383a61b

Reply:
Oh yeah, you're right - the PR hid the context, I should've checked more closely. And frankly, I should probably run through the whole document more closely.